public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,96 @@
+   You will be able to back and edit others settings later.
+
+5. From the left sidebar menu select the **APIs & Services** -> **Credentials**
+   and click the **Create credentials** button to select the **OAuth client ID**
+   -> **Other** -> click **Create** to proceed with a default suggested **Name**
+   for your new **OAuth client ID**.
+
+   ![](/static/img/gdrive-create-credentials.png)
+
+6. The new generated **client ID** and **client secret** should be showed to
+   you. Use them for further DVC's configuration.
+
+```dvc
+$ dvc remote add -d myremote gdrive://path # See URL format section below
+$ dvc remote modify myremote gdrive_client_id my_gdrive_client_id
+$ dvc remote modify myremote gdrive_client_secret gdrive_client_secret
+```
+
+⚠️ It is safe to share a **client ID** and a **client secret** at least with
+your teammates or inside an organization. These credentialds are used to
+[retrieve an authorization token](https://developers.google.com/identity/protocols/OAuth2)
+and
+[authenticate your application](https://developers.google.com/drive/api/v2/about-auth)
+to Google. Credentials don't give an access to your Google Drive storage. But
+Google limits global number of queries per each **client ID** and uncontrolled
+usage of your **client ID** by outer world can lead to
+[reaching the allowed Google API usage limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#quota).
+
+## Google Drive remote URL format
+
+A remote URL can contain **root** which is an alias of your topmost Google Drive
+directory. To point the remote to the path under a Google Drive's **root**
+directory:
+
+```dvc
+$ dvc remote add -d myremote gdrive://root/path/to/dir
+```
+
+As alternative, you can obtain unique **Google Drive id** of your desired remote
+directory and use it as **base** remote path in URL:
+
+```dvc
+$ dvc remote add -d myremote gdrive://0AIac4JZqHhKmUk9PDA/path/to/dir
+```
+
+To obtain the **Google Drive ID** of a remote directory navigate inside that
+directory in your web browser with opened Google Drive and find the **Google
+Drive ID** in address bar. For example, for the URL
+
+```dvc
+https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA
+```
+
+the **ID** will be equal to `0AIac4JZqHhKmUk9PDA`.
+
+**Shared Team Drive** can be referenced by **Google Drive ID** in the similar
+way.
+
+## Authorization
+
+On the first usage of the remote you will be prompted to visit an access token
+generation link in browser. It will ask you to log into the Google account
+associated with Google Drive, which you want to use as DVC's remote. The login
+process will guide you through granting Google Drive access permissions to the
+used Google Project.
+
+On successful access token generation a token data will be cached in the git
+ignored directory under the path **.dvc/tmp/gdrive-user-credentials.json**.
+
+⚠️ Do not share an access token data with anyone else to prevent an unauthorized
+access to your Google Drive.
```

Comment:
How is this different to the credentials mentioned earlier, which are safe to share?

"The mandatory OAuth credentials, **client ID** and **client secret**, can be generated only once. Please share them with other DVC users in your team to grant them access to Google's API."

Reply:
They are just different credentials, different files, different names, different security policies. Probably, if it's not that clear we should make it even more explicit.

Reply:
Yeah I thin it's confusing, could seem like its contradictory. I'll try to make it clearer.

Reply:
Done in 128b95ca.